targetconfig.json

```diff
@@ -5,11 +5,20 @@
             "LEGO"
         ],
         "approvedRepos": [
-            "microsoft/pxt-automation"
+            "microsoft/pxt-automation",
+            "THEb0nny/HiTechnic-Color-Sensor-v2"
         ],
         "preferredRepos": [
             "microsoft/pxt-automation"
-        ]
+        ],
+        "approvedRepoLib": {
+            "microsoft/pxt-automation": {
+                "preferred": true
+            },
+            "THEb0nny/HiTechnic-Color-Sensor-v2": {
```

Comment:
please separate this to another pr, extensions need their own review before approving; in this case, looks like it needs docs for all blocks, and an icon for the extension to show up in the page. Here's an example of an extension for minecraft that was just added by a team member in case it helps: https://github.com/microsoft/makecode-minecraft-music

In addition, note that fields in approvedRepoLib must be lowercase; I'll run the build in a second but it should throw an error on that for now

Reply:
I removed all this and will make a separate pr as you said.

Reply:
I will also add all changes to the extension for ht sensors to the repository.